mitosheet/src/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -122,101 +124,78 @@ const GraphSidebar = (props: {
     columnDtypesMap: Record<string, string>;
```

Comment:
There's an unused thing called `GraphObject` at the top of this file we can remove, I think